pkg/logger/gorm.go

```diff
@@ -1,95 +1,68 @@
-		affectedRows := values[5].(int64)
-
-		logger := gl.logger.WithFields(Fields{
-			"sql": Fields{
-				"duration":      duration,
-				"affected_rows": affectedRows,
-				"file_location": location,
-				"values":        formattedValues,
-			},
-		})
+func (g gormLogger) LogMode(logger.LogLevel) logger.Interface {
+	// we ignore changes to the log level.
+	// making changes to logger level causes a change to the root logger.
+	return g
+}
 
-		logger.Debugf(sql)
-	} else {
-		gl.logger.Tracef("%v", values[2:]...)
-	}
+func (g gormLogger) Info(ctx context.Context, msg string, args ...interface{}) {
+	g.logger.Infof(msg, args...)
 }
 
-// Code blatantly stolen and modified for our needs from Gorm:
-// https://github.com/jinzhu/gorm/blob/2a3ab99/logger.go#L30-L109
-func formatValues(values []interface{}) string {
-	var formattedValues []string
-	for _, value := range values {
-		indirectValue := reflect.Indirect(reflect.ValueOf(value))
-		if indirectValue.IsValid() {
-			value = indirectValue.Interface()
-			if t, ok := value.(time.Time); ok {
-				if t.IsZero() {
-					formattedValues = append(formattedValues, fmt.Sprintf("'%v'", "0000-00-00 00:00:00"))
-				} else {
-					formattedValues = append(formattedValues, fmt.Sprintf("'%v'", t.Format("2006-01-02 15:04:05")))
-				}
-			} else if b, ok := value.([]byte); ok {
-				if str := string(b); isPrintable(str) {
-					formattedValues = append(formattedValues, fmt.Sprintf("'%v'", str))
-				} else {
-					formattedValues = append(formattedValues, "'<binary>'")
-				}
-			} else if r, ok := value.(driver.Valuer); ok {
-				if value, err := r.Value(); err == nil && value != nil {
-					formattedValues = append(formattedValues, fmt.Sprintf("'%v'", value))
-				} else {
-					formattedValues = append(formattedValues, "NULL")
-				}
-			} else {
-				switch value.(type) {
-				case int, int8, int16, int32, int64, uint, uint8, uint16, uint32, uint64, float32, float64, bool:
-					formattedValues = append(formattedValues, fmt.Sprintf("%v", value))
-				default:
-					formattedValues = append(formattedValues, fmt.Sprintf("'%v'", value))
-				}
-			}
-		} else {
-			formattedValues = append(formattedValues, "NULL")
-		}
-	}
+func (g gormLogger) Warn(ctx context.Context, msg string, args ...interface{}) {
+	g.logger.Warnf(msg, args...)
+}
 
-	return strings.Join(formattedValues, " ")
+func (g gormLogger) Error(ctx context.Context, msg string, args ...interface{}) {
+	g.logger.Errorf(msg, args...)
 }
 
-func isPrintable(s string) bool {
-	for _, r := range s {
-		if !unicode.IsPrint(r) {
-			return false
+func (g gormLogger) Trace(ctx context.Context, begin time.Time, fc func() (string, int64), err error) {
+	var fields Fields
+
+	if g.logger.Level() >= logrus.DebugLevel {
+		sql, rows := fc()
+
+		fields = Fields{
+			gormTraceFieldKey: Fields{
+				"elapsed_miliseconds": time.Since(begin).Seconds() * 1000,
+				"affected_rows":       rows,
+				"sql":                 sql,
```

Comment:
Perhaps is better to avoid logging `sql` at all as logging full [SQL is considered as a bug](https://github.com/go-gorm/gorm/pull/5288). Maybe we should create a follow-up Jira to include `sql` statement in the log entry once the issue is fixed in gorm, wdyt?
/cc @fotos

Reply:
That looks like a regression in `gorm`? 😢 

We definitely need parameterized query logging instead of logging the full SQL.

I'm OK with skipping the SQL statement for now and creating a follow up Jira.

Reply:
It was resolved by implementing `ParamsFilter` [interface](https://github.com/go-gorm/gorm/blob/ddd3cc2502eb0a0193e10ec6360d5e83d19493a8/interfaces.go#L29).
See the [fix commit](https://github.com/go-gorm/gorm/commit/ddd3cc2502eb0a0193e10ec6360d5e83d19493a8) by gorm.